doc/user-guides/multi-cluster.md

```diff
@@ -0,0 +1,602 @@
+
+We are using curl to hit our endpoint. As we are using letsencrypt staging in this example we pass the `-k` flag.
+
+```bash
+curl -k -w "%{http_code}" https://$(kubectl get httproute api -n ${gatewayNS} -o=jsonpath='{.spec.hostnames[0]}')
+```
+
+We should see a `403` response. With our gateway and policies in place we can now allow other teams to use the gateway:
+
+```bash
+kubectl patch gateway ${gatewayName} -n ${gatewayNS} --type='json' -p='[{"op": "replace", "path": "/spec/listeners/0/allowedRoutes/namespaces/from", "value":"All"}]'
+```
+
+### Extending this Gateway to multiple clusters and configuring GEO based routing
+
+In order to have this gateway distributed across multiple clusters, we would follow the above instructions for each cluster as noted at the start. By default that would set up a `RoundRobin` DNS strategy to bring traffic to the different clusters. Enabling our gateways to serve clients based on their GEO is relatively straight forward based on our current configuration.
+
+Assuming you have deployed a gateway instance to multiple clusters and configured it based on this document. The next step is to inform the DNS controller about what Geographic region the gateways it can see are in.
+
+So for example if we have a cluster in North America and a Cluster in the EU we can bring traffic to those gateways based on location simply by applying the following label:
+
+In our North American cluster:
+
+```bash
+kubectl label --overwrite gateway ${gatewayName} kuadrant.io/lb-attribute-geo-code=US -n ${gatewayNS}
+```
+
+In our European Cluster
+
+```bash
+kubectl label --overwrite gateway ${gatewayName} kuadrant.io/lb-attribute-geo-code=EU -n ${gatewayNS}
+```
+
+
+After some time you can check the geo distribution using the HTTPRoute host `kubectl get httproute api -n ${gatewayNS} -o=jsonpath='{.spec.hostnames[0]}'` via site such as https://dnsmap.io/
+
+## Developer (WIP)
+
+For this part of the walkthrough, we will go through leveraging an Open API Spec (OAS) to define an API and also using the powerful kuadrant OAS extensions to define the routing and the auth and rate limiting requirements. We will then use the `kuadrantctl` tool to generate an AuthPolicy, a HTTPRoute and  a RateLimitPolicy to enforce what is in our OAS. While we use the `kuadrantctl` tool here, it is worth noting that it is not essential. AuthPolicy, RateLimitPolicy and HTTPRoutes can be created independently.
+
+### Pre Req
+
+- Install kuadrantctl. You can find a compatible binary and download it from the [kuadrantctl releases page](https://github.com/kuadrant/kuadrantctl/releases )
+- Ability to distribute resources to the 2 or more clusters as per platform engineer.
+
+### Setup HTTPRoute and backend
+
+Copy the following examples to a local location:
+[sample OAS API Key spec](../../examples/oas-apikey.yaml)
+[sample OAS OIDC spec](../../examples/oas-oidc.yaml)
+
+setup some new env vars:
+
+```
+export oasPath=/path/to/oas.yaml
```

Comment:
This export is inside the snipped clipboard, but it can't be copied and used. My suggestion is to ensure user that he must be located in kuadrant-operator project and then change this to `pwd` or similar aproach.

Reply:
in the doc we ask them to copy the oas example to their local machine so this is the path to that copy